pkg/storage/factory.go

```diff
@@ -105,7 +105,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&cfg.MaxParallelGetChunk, "store.max-parallel-get-chunk", 150, "Maximum number of parallel chunk reads.")
 	cfg.BoltDBShipperConfig.RegisterFlags(f)
 	f.IntVar(&cfg.MaxChunkBatchSize, "store.max-chunk-batch-size", 50, "The maximum number of chunks to fetch per batch.")
-	cfg.TSDBShipperConfig.RegisterFlagsWithPrefix("tsdb.", f)
+	cfg.TSDBShipperConfig.RegisterFlagsWithPrefix("tsdb", f)
```

Comment:
shouldn't the additional dot be removed from the `(*TSDBShipperConfig)RegisterFlagsWithPrefix` definition? if you remove from here instead of there, if prefix is "" the flag will start with a period which looks weird

Reply:
the main reason why I've not done it at first was because `boltdb` prefix does not have the dot so I've aligned the both of them. 
The PR has been amended to address your comment.